### PR TITLE
Detector Manager Monitoring

### PR DESCRIPTION
**Pull request by ankgupta**

### CODE REVIEW

File: anomdetect/src/test/java/com/expedia/adaptivealerting/anomdetect/detect/outlier/algo/cusum/CusumDetectorTest.java [hunk 1]
```diff
@@ -145,12 +146,31 @@ public void testClassify_twoTailed() {
...
+    }
+
+
     private void testClassify(CusumDetectorParams params, AnomalyType anomalyType, CusumDetectorTestRow[] testRows, boolean trusted) {
         val detector = new CusumDetector(detectorUuid, params, trusted);
         assertEquals(detectorUuid, detector.getUuid());
         assertSame(params, detector.getParams());
 
-        // FIXME Hack to handle an off-by-one bug in the CusumDetector. [WLW]
+        // FIXME Hack to handle an off-by-one bu g in the CusumDetector. [WLW]
```
Comment (williewheeler) [inline anomdetect/src/test/java/com/expedia/adaptivealerting/anomdetect/detect/outlier/algo/cusum/CusumDetectorTest.java]: It was better without the space :D

Comment (ankgupta) [reply anomdetect/src/test/java/com/expedia/adaptivealerting/anomdetect/detect/outlier/algo/cusum/CusumDetectorTest.java]: Fixed

File: anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/detect/breakout/algo/edmx/EdmxDetector.java [hunk 2]
```diff
@@ -32,6 +32,7 @@
 
 @Slf4j
 public final class EdmxDetector implements BreakoutDetector {
+    private final String NAME = "edmx";
```
Comment (williewheeler) [inline anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/detect/breakout/algo/edmx/EdmxDetector.java]: If we do it this way (and fine by me), wouldn't we want this to be `static` as well?

File: anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/detect/outlier/algo/constant/ConstantThresholdDetector.java [hunk 3]
```diff
@@ -33,6 +33,7 @@
  */
 @ToString(callSuper = true)
 public final class ConstantThresholdDetector extends AbstractOutlierDetector {
+    private final String NAME = "constant-threshold";
```
Comment (williewheeler) [inline anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/detect/outlier/algo/constant/ConstantThresholdDetector.java]: Simliar remark as above re `static`.

File: anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/detect/outlier/algo/cusum/CusumDetector.java [hunk 4]
```diff
@@ -43,6 +43,7 @@
  */
 @ToString(callSuper = true)
 public final class CusumDetector extends AbstractOutlierDetector {
+    final private String NAME = "cusum";
```
Comment (williewheeler) [inline anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/detect/outlier/algo/cusum/CusumDetector.java]: Please use consistent modifier order.

File: anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/detect/outlier/algo/forecasting/ForecastingDetector.java [hunk 5]
```diff
@@ -53,7 +53,7 @@
  */
 @ToString(callSuper = true)
 public final class ForecastingDetector extends AbstractOutlierDetector {
-
+    final private String NAME;
```
Comment (williewheeler) [inline anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/detect/outlier/algo/forecasting/ForecastingDetector.java]: See above re: modifier order

**Pull request merged**